scripts/ft-benchmark.sh

```diff
@@ -8,6 +8,7 @@ date
 # Otherwise nearup and cargo don't work even if installed properly
 PATH=/home/ubuntu/.local/bin/:$PATH
 export PATH=$PATH:$HOME/.cargo/bin
+source benchmarks/continous/db/tool/dbprofile
```

Comment:
In order to not force a specific working directory on the script, I would suggest to make this path relative to the path of this script. You can get that with `dirname "$0"`.